frontend/ui/impl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -155,6 +165,9 @@ public void after(java.util.List<? extends VFileEvent> events) {
         for (val event : events) {
           if (event instanceof VFileContentChangeEvent) {
             if (((VFileContentChangeEvent) event).getFile().getFileType().getName().equals(FileTypeIds.NAME)) {
+              if (slideInNotification != null && !slideInNotification.isExpired()) {
+                slideInNotification.expire();
```

Comment:
I suppose that this `after` method executes on UI thread right? Pls check, and mark `after` as `@UIEffect` in stubs if so.

`com.intellij.openapi.vfs.newvfs.BulkFileListener` -> `A listener for VFS events, invoked inside write-action.`

Reply:
yes, it is run on UI thread